examples/cnn_inference_f16.cpp

```diff
@@ -0,0 +1,806 @@
+/*******************************************************************************
```

Comment:
I think adding a new example for f16 data type is redundant as the only difference between `cnn_inference_f16.cpp` and `cnn_inference_f32.cpp` would be using different data types `f16` vs `f32`. It looks like it's not worth having the whole new example.

Reply:
Thanks for the review, I try to code a more reasonable example about f16.

Reply:
I deleted this file, I think that we need a new PR to rework the cnn_inference_example for f16/bf16/f32 data type.

Reply:
Ok, just keep in mind that we usually provide examples to demonstrate use of our API, though it's not a strict requirement.
A an example, we provide examples f32 and int8, the difference is that for int8 we demonstrate use of our output scale API, that is necessary for int8 data types.